lbjava/pom.xml

```diff
@@ -29,7 +29,6 @@
 			<groupId>gurobi</groupId>
 			<artifactId>gurobi</artifactId>
 			<version>5.0.1</version>
-			<optional>true</optional>
```

Comment:
I made gurobi optional. When I deleted `gurobi` from my local .m2 repo and ran the compile step, maven downloaded the gurobi package from CogComp repo. But the test on semaphore didn't download and it failed. https://semaphoreci.com/danyaljj/lbjava/branches/pull-request-42/builds/11